### DISCUSSION TITLE
oldwinter.top主发布站将自动重定向至garden.oldwinter.top备发布站

### DISCUSSION THREAD

**Discussion by oldwinter** [oldwinter/knowledge-garden#17] [Announcements]
📢之前我买了obsidian的发布服务6个月，做为主发布站，即https://oldwinter.top ，现在到期了已经删除啦😅。
目前，我将将其自动重定向了https://garden.oldwinter.top ，使用的是开源的quartz免费方案。若需要看oldwinter.top中原文章的朋友，界面右上角搜索笔记或文章的关键词即可找到。笔记内容是完全一致的，不会有任何删除。

**Comment by chinyajie** [comment]
quartz 的更新还挺频繁的，后期优化应该是有保障的。只不过受限于 Hugo 只能把回连放到底部而不是右侧，以及不支持目录访问有点可惜。另外请教您个问题，backlink 栏除了标题，想放入少量文字摘要好实现吗，我看到官方的案例中有一个是实现了的。

**Reply by oldwinter** [reply]
是的，作者一直在迭代，discord里面大家也挺活跃的。不妨把那个带摘要的案例链接贴出来瞅瞅？我感觉不需要文字摘要对我而言足够啦，所以就没去折腾了。

**Reply by chinyajie** [reply]
https://scalingsynthesis.com/Q-How-might-we-apply-map-filter-reduce-to-notes-and-what-other-primitives-are-relevant-to-this-domain/
